File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;

        for (String word : babbling) {
            word = word.replaceAll("aya|woo|ye|ma", "");
            if (word.isEmpty()) {
                answer++;
            }
        }

        return answer;
    }
}

File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.kt
class Solution {
    fun solution(babbling: Array<String>): Int {
        var answer = 0

        for (word in babbling) {
            var newWord = word.replace(Regex("aya|woo|ye|ma"), "")
            if (newWord.isEmpty()) {
                answer++
            }
        }

        return answer
    }
}
